readme.md

```diff
@@ -165,6 +166,8 @@ Software tools for analyzing embedded/IoT devices and firmware.
 
 <!--lint ignore no-repeat-punctuation-->
 
+
+- [IoT Security@CERN](https://doi.org/10.5281/zenodo.1035034) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1035034.svg)](https://doi.org/10.5281/zenodo.1035034)
```

Comment:
Remove badge as the description. If you wish you can add a textual description. However make sure you'll finish the sentence with a '.'.

Reply:
Also I'd like to keep the case studies in alphabetical order as for now